tests/karate/src/test/java/common/net/MultiMsgWebSocketClient.java

```diff
@@ -13,15 +13,15 @@ public class MultiMsgWebSocketClient extends WebSocketClient {
 
   private final MessageQueue queue;
   private final Logger logger;
-  private JsonConverter jsonConverter;
-  private final String nonAttendeePk = "oKHk3AivbpNXk_SfFcHDaVHcCcY8IBfHE7auXJ7h4ms=";
-  private final String nonAttendeeSkHex = "0cf511d2fe4c20bebb6bd51c1a7ce973d22de33d712ddf5f69a92d99e879363b";
+  private JsonConverter jsonConverter = new JsonConverter();
+  private static final String nonAttendeePk = "oKHk3AivbpNXk_SfFcHDaVHcCcY8IBfHE7auXJ7h4ms=";
+  private static final String nonAttendeeSkHex = "0cf511d2fe4c20bebb6bd51c1a7ce973d22de33d712ddf5f69a92d99e879363b";
+  private int idAssociatedWithSentMessage;
```

Comment:
What if we send multiple messages ?

How would you handle this case with the current API ?

Reply:
Yes you are right, I applied some changes